### ISSUE TITLE
Opposite of lsp-install-server

### ISSUE THREAD

**Issue by raxod502** [emacs-lsp/lsp-mode#3503]
Is there any interest in a command `M-x lsp-uninstall-server` which would do the opposite of `M-x lsp-install-server`? I think the alternative way of uninstalling a language server is to manually root around in `~/.emacs.d` and guess which directory to delete, which seems like a subpar user experience.

**Comment by VisionaryAppDev** [comment]
If you use Doom Emacs, there is an uninstall command like this one:

![image](https://user-images.githubusercontent.com/27724011/166411839-0cebe7b0-0941-426e-8c76-039a102d49ff.png)

**Comment by ericdallo** [comment]
I think it makes sense to have a command for that, WDYT @yyoncho?

**Comment by yyoncho** [comment]
I guess we can, although I don't see big value in it because you can go in lsp install dir and delete whatever your want or you can use lsp-disabled-clients to stop the servers that you don't want to run

**Comment by ericdallo** [comment]
I think it's more like a way to clean what lsp-mode did, just to have a way in and out

**Comment by yyoncho** [comment]
> I think it's more like a way to clean what lsp-mode did, just to have a way in and out

I see that but in the end, you can have M-x lsp-unsinstall-server and after that, you may still hit the server because you may have the package installed globally. It is not like in vscode where the server is bundled with the extension.

**Comment by ericdallo** [comment]
yes, but at least we would uninstall anything installed via lsp-mode, I think it's expected to lsp-mode don't touch global (outside emacs) installs. Maybe we can make it clear on command docstring
